crates/astria-proto/src/lib.rs

```diff
@@ -1,3 +1,7 @@
 pub mod execution {
     include!(concat!(env!("OUT_DIR"), "/astria.execution.v1.rs"));
 }
+
+pub mod sequencer_relayer {
```

Comment:
This related to a comment I left in #7: we lose part of the name here. I think we shouldn't drop the `v1`:
```rust
pub mod sequencer_relayer {
    pub mod v1 {
        include!(...);
    }
}
```